build.gradle

```diff
@@ -14,16 +21,16 @@ repositories {
     maven { url 'https://jitpack.io' }
 }
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+sourceCompatibility = '11'
+targetCompatibility = '11'
 
 def versionsMap = ['IU-2021.1':'211.6693.108', 'IU-2021.2':'212.4746.57', 'IU-2021.3':'213.5744.122', 'IU-2022.1':'221.5080.1', 'IU-2022.2':'222.3345.16', 'IU-2022.3':'223.7571.175']
 
 intellij {
     version = ideaVersion //for a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     pluginName = 'com.redhat.devtools.intellij.tekton'
     // use '<git repo clone>/build/idea-sandbox/plugins/Telemetry by Red Hat' if working from source
-    plugins = ['terminal', 'yaml', 'markdown', 'com.intellij.kubernetes:' + versionsMap[ideaVersion], "com.redhat.devtools.intellij.telemetry:0.0.3.33", 'com.redhat.devtools.intellij.kubernetes:0.6.2.179']
+    plugins = ['terminal', 'yaml', 'markdown', "com.redhat.devtools.intellij.telemetry:0.0.3.33", 'com.intellij.kubernetes:' + versionsMap[ideaVersion], '/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat']
```

Comment:
replace `'/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat'` by the intellij-kubernetes plugin release once it's published.